src/apps/code-editor/src/app/components/FileActions/components/Delete/Delete.js

```diff
@@ -22,14 +22,18 @@ export const Delete = React.memo(function Delete(props) {
 
   return (
     <div className={styles.DeleteBtn}>
-      <Button
-        kind="warn"
-        onClick={() => setOpen(true)}
-        className={styles.Button}
-      >
-        <FontAwesomeIcon icon={faExclamationCircle} />
-        Delete
-      </Button>
+      {props.fileName !== "loader" ? (
```

Comment:
`fileName` is a user controlled value and as such can be brittle when relying on it. Lets use the `type`, which should also be `loader` for this specific view, as that is a system controlled value.

Reply:
@shrunyan props is showing  fileType: "views"  for loader view, thats why I went with fileName.